journal/updated/01-journal.adoc

```diff
@@ -440,7 +445,7 @@ This necessity being laid upon me, I spoke to my master in that dialect;
 he was not offended at it, because he was convinced of the truth of it,
 and that it ought to be spoken to everyone. But when I spoke it to my mistress,
 she took a stick and gave me such a blow upon my bare head,
-that made it swell and sore for a considerable time. She was so disturbed at it,
+that made it swell and was sore for a considerable time. She was so disturbed at it,
```

Comment:
this still sounds weird to me. I think it should say: she took a stick and gave me such a blow upon my bare head,
**that it swelled and was sore** for a considerable time.